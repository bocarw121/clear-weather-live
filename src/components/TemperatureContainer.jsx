import React from "react";
import { StyleSheet, View } from "react-native";

const TemperatureContainer = ({ children, type }) => {
  const containerStyle = [styles.container];

  type === "current"
    ? containerStyle.push(styles.current)
    : type === "hourly"
    ? containerStyle.push(styles.hourly)
    : type === "weekly"
    ? containerStyle.push(styles.weekly)
    : null;

  return <View style={containerStyle}>{children}</View>;
};

export default TemperatureContainer;

const styles = StyleSheet.create({
  container: {
    justifyContent: "center",
    alignItems: "center",
  },
  current: {},
  hourly: {},
  weekly: {},
});
