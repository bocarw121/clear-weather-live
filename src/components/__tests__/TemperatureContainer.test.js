import React from "react";

import { render } from "@testing-library/react-native";

import TemperatureContainer from "../TemperatureContainer";

it("Should render the TemperatureContainer", () => {
  const screen = render(<TemperatureContainer />);

  expect(screen).toMatchSnapshot();
});
