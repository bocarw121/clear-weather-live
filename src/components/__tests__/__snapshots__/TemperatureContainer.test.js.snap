// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the TemperatureContainer 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
      },
    ]
  }
/>
`;
